README.md

```diff
@@ -116,6 +117,22 @@ Rendering RTCView.
 
 ### Custom APIs
 
+#### registerGlobals()
+
+By calling this method the JavaScript global namespace gets "polluted" with the following additions:
+
+* `navigator.getUserMedia()` (deprecated W3 API)
```

Comment:
Let's not add this one, since we don't support the callback API and some shims may think we do because we support thie deprecated method.

Reply:
Done.